MekHQ/src/mekhq/gui/dialog/GMToolsDialog.java

```diff
@@ -413,6 +451,151 @@ private JPanel getCallsignGenerator() {
+        for (Clan clan : clans) {
+            originClanModel.addElement(clan.getFullName(bloodnameYear));
+        }
+        originClanPicker = new JComboBox<>(originClanModel);
+        originClanPicker.setName("originClanPicker");
+        originClanPicker.setSelectedIndex(0);
+        originClanPicker.addActionListener(evt -> validateBloodnameInput());
+        namePanel.add(originClanPicker, newGridBagConstraints(gridx++, gridy + 1));
+
+        JLabel bloodnameEraLabel = new JLabel(resources.getString("bloodnameEraLabel.text"));
+        bloodnameEraLabel.setName("bloodnameEraLabel");
+        namePanel.add(bloodnameEraLabel, newGridBagConstraints(gridx, gridy));
+
+        bloodnameEraPicker = new JComboBox<>(ERAS);
+        bloodnameEraPicker.setName("bloodnameEraPicker");
+        bloodnameEraPicker.setSelectedIndex(0);
+        bloodnameEraPicker.addActionListener(evt -> validateBloodnameInput());
+        namePanel.add(bloodnameEraPicker, newGridBagConstraints(gridx++, gridy + 1));
+
+        JLabel phenotypeLabel = new JLabel(resources.getString("phenotypeLabel.text"));
+        phenotypeLabel.setName("phenotypeLabel");
+        namePanel.add(phenotypeLabel, newGridBagConstraints(gridx, gridy));
+
+        DefaultComboBoxModel<Phenotype> phenotypeModel = new DefaultComboBoxModel<>();
+        phenotypeModel.addElement(Phenotype.GENERAL);
+        for (Phenotype phenotype : Phenotype.getExternalPhenotypes()) {
+            phenotypeModel.addElement(phenotype);
+        }
+        phenotypePicker = new JComboBox<>(phenotypeModel);
+        phenotypePicker.setName("phenotypePicker");
+        phenotypePicker.setSelectedItem(Phenotype.GENERAL);
+        phenotypePicker.setRenderer(new DefaultListCellRenderer() {
+            @Override
+            public Component getListCellRendererComponent(JList<?> list, Object value, int index,
+                                                          boolean isSelected, boolean cellHasFocus) {
+                setOpaque(list.isOpaque());
+                if (isSelected) {
+                    setBackground(list.getSelectionBackground());
+                    setForeground(list.getSelectionForeground());
+                } else {
+                    setBackground(list.getBackground());
+                    setForeground(list.getForeground());
+                }
+                setFont(list.getFont());
+
+                setText((value == null) ? ""
+                        : (value instanceof Phenotype ? ((Phenotype) value).getGroupingName() : "ERROR"));
+
+                setEnabled(list.isEnabled());
+
+                if (cellHasFocus) {
+                    Border border = null;
+                    if (isSelected) {
+                        border = DefaultLookup.getBorder(this, ui, "List.focusSelectedCellHighlightBorder");
+                    }
+                    if (border == null) {
+                        border = DefaultLookup.getBorder(this, ui, "List.focusCellHighlightBorder");
```

Comment:
I get errors when building with `DefaultLookup.*` under JDK11. Perhaps this should move to [`UIManager`](https://docs.oracle.com/javase/8/docs/api/javax/swing/UIManager.html)?

Reply:
Fixed by reworking how the override works